tests/peer/meshRoom.js

```diff
@@ -274,6 +274,18 @@ describe('MeshRoom', () => {
         assert.equal(answerSpy.callCount, 1);
         assert(answerSpy.calledWith(meshRoom._localStream));
       });
+
+      it('should not create MediaConnection when has a connection with peer and peerId is larger', () => {
```

Comment:
Could you add several test cases to provide living documentation?
- A `peerID` shorter/longer than the other
- A `peerID` started with underscores
- etc...